bazel-mode.el

```diff
@@ -76,7 +88,8 @@
   (setq-local comment-start-skip "#+")
   (setq-local comment-end "")
   (setq-local comment-use-syntax t)
-  (setq-local font-lock-defaults '(nil)))
+  (setq-local font-lock-defaults '(nil))
+  (add-hook 'before-save-hook 'bazel-mode--buildifier-before-save-hook))
```

Comment:
nit: use `#'` to refer to function symbols so that the byte compiler complains if the function isn't defined.

Reply:
Done.